commands/service_execute.go

```diff
@@ -105,11 +105,20 @@ func (c *serviceExecuteCmd) runE(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	fmt.Printf("Task %s returned output %s with data:\n%s\n",
-		pretty.Success(c.taskKey),
-		pretty.Colorize(pretty.FgCyan, result.OutputKey),
-		pretty.ColorizeJSON(pretty.FgCyan, nil, true, []byte(result.OutputData)),
-	)
+	errResult := result.GetError()
+	outputResult := result.GetOutput()
+	if errResult != nil {
+		fmt.Printf("Task %s cannot be completed, error:\n%s\n",
```

Comment:
Put a fail sign and remove colors to keep consistency with others commands printf

Reply:
done